db/data_migration/20200817081009_add_all_nation_applicability_to_editions.rb

```diff
@@ -0,0 +1,14 @@
+PUBLISHED_AND_PUBLISHABLE_STATES = %w[published draft archived submitted rejected scheduled].freeze
+EDITIONS_WITH_NATIONAL_APPLICABILITY = ['DetailedGuide', 'Publication', 'Consultation'].freeze
+
+edition_scope = Edition.where(type: EDITIONS_WITH_NATIONAL_APPLICABILITY).where(state: PUBLISHED_AND_PUBLISHABLE_STATES)
+
+def update_all_nation_applicability(edition)
+  if edition.nation_inapplicabilities.any?
+    edition.update_column(:all_nation_applicability, false)
+  else
+    edition.update_column(:all_nation_applicability, true)
```

Comment:
I think we could skip this as we have `default: true` and that might save some time on this migration?